### PR TITLE
Information about scoping PRs

### PR DESCRIPTION
**Pull request by sebasslash**
Updating the contributing docs to give guidance on scoping PRs.

### CODE REVIEW

**Review by annawinkler — DISMISSED** [review]
🌟 I like this addition to our contributing guidelines. Thank you for adding it!

**Review by Uk1288 — APPROVED** [review]

**Review by brandonc — DISMISSED** [review]
> If submitting multiple PRs, please ensure to mention which PR depends on which in order to help the reviewers. 

I only have some problems with this statement. It's sometimes painful to open multiple, dependent PRs because you have to base one branch on another branch (which this section does not mention) and keeping them up to date and properly rebased, as well as roll-up-merging multiple approvals can make both authors' and reviewers' jobs more difficult. If there are multiple PRs with dependencies, they should sometimes be combined! The only reason why they shouldn't be is when trying to parallelize the effort or you can author and merge the shared dependencies first.

I think there is enough nuance in there to not apply a blanket statement to the contributing docs like this. What specific problem is this trying to address?

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ If you are making relevant changes that is worth communicating to our users, ple
...
 
+### Scoping pull requests that add new resources 
+
+There are instances where several new resources being added (i.e Workspace Run Tasks and Organization Run Tasks) are coalesced into one PR. In order to keep the review process as efficient and least error prone as possible, we ask that you please scope each PR to an individual resource even if the multiple resources you're adding share similarities. If joining multiple related PRs into one single PR makes more sense logistically, we'd ask that you organize your commit history by resource. A general convention for this repository is one commit for the implementation of the resource's methods, one for the integration test, and one for cleanup and housekeeping (e.g modifying the changelog/docs, generating mocks, etc).   
+
+**Note HashiCorp Employees Only:** When submitting a new set of endpoints please ensure that one of your respective team members approves the changes as well before merging.
```
Comment (annawinkler) [inline docs/CONTRIBUTING.md:17]: What do you think about adding this request to the PR template also? (As a checklist item, perhaps?)

**Review by annawinkler — APPROVED** [review]
⭐

**Pull request merged**